cmd/thanos/store.go

```diff
@@ -50,11 +51,20 @@ func registerStore(m map[string]setupFunc, app *kingpin.Application, name string
 	blockSyncConcurrency := cmd.Flag("block-sync-concurrency", "Number of goroutines to use when syncing blocks from object storage.").
 		Default("20").Int()
 
+	selectorLabels := cmd.Flag("selector-label", "Store Gateway selector labels that will be exposed in info endpoint (repeated).").
```

Comment:
Let's make sure this is the same, consistent for `receive` and maybe `rule`? Querier either dropped it already or will drop as we aggregate labels there from storeAPIs.